float32 presa_x
float32 presa_y
float32 presa_theta
float32 depredador_x
float32 depredador_y
float32 depredador_theta
---
bool success
